tools/fluent/README.md

```diff
@@ -0,0 +1,11 @@
+# Fluent for localizers
+
+<!-- toc -->
+
+Fluent is a localization paradigm, designed at Mozilla [as part of L20n](https://github.com/projectfluent/fluent/wiki/Fluent-and-L20n), to unleash the expressive power of the natural language. The format used to describe translation resources used by Fluent is called `FTL`.
```

Comment:
Uhm, had no idea. Should we drop the L20n paragraph entirely?

Reply:
Some localizers might still know Fluent as L20n. It might be good to mention it; something like `previously known as L20n`.

There's also https://github.com/projectfluent/fluent/issues/57 which proposes to call Fluent a _system_, not a _paradigm_ etc.

Reply:
Re-reading, that comma before the *to* is weird. This?

```
Fluent is a localization system, previously known as L20n, designed at Mozilla to unleash the expressive power of the natural language.
```

Reply:
LGTM!